### ISSUE TITLE
make footprints compatible with ethereal by mistere

### ISSUE THREAD

**Issue by MeseCraft** [MeseCraft/MeseCraft#19]
MisterE123 commented 1 year ago
here is a fork of footprints that adds ethereal dirt compatibility:

https://notabug.org/CenteriaSurvival/footprints

just replace the mod

MeseCraft commented 11 months agoOwner 
Awesome. Yeah, there are lots of integrations that are not fully implemented yet. I'll checkout your update with the added support for ethereal. This will eventually need to be done with things that have ground soft nodes. farming, cave_realms, df_caverns, etc. It'll get added.

**Comment by MisterE123** [comment]
do you even still have ethereal in the game?

**Comment by MeseCraft** [comment]
no, only a few things from it like leaves transparent and ice/water fire things
